common/src/lib.rs

```diff
@@ -41,6 +41,21 @@ pub trait AstNode: Debug + PartialEq + Clone + Spanned + Serialize {
     const TYPE: &'static str;
 }
 
+/// Target runtime environment.
+#[derive(Debug, Serialize, Deserialize)]
+pub enum TargetEnv {
```

Comment:
I'm a bit worried about adding this to `swc_common`. Ideally, only `node` and `browser` could be the only two target supported by ecmascript, but `swc_common` is for language-agnostic stuff.
Can you move this to `swc_ecma_ast`?

Reasoning is that, I may implement support for other targets but ecmascript does not support such target. I'm not sure if such case exist, but I want to stay safe.

Reply:
Thanks, I wasn't sure where to put this so that's helpful. It is moved to `swc_ecma_ast` now :+1: